### ISSUE TITLE
Using the Cookie pre-defined hidden field value passes "undefined" if the cookie isn't set

### ISSUE THREAD

**Issue by Mosnar** [verbb/formie#536]
**Description**

If I set a hidden field's default value to pull from a cookie and that cookie doesn't exist, it appears to be using "undefined" as the value in the submission. 

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes 
- Custom Form Templates: Partial

**Additional info**
- Plugin version: 1.4.12
- Craft version: 3.7.7 Pro
- Multi-site: No

**Additional context**
PHP 8 on Amazon Linux 2

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.12",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.13](https://github.com/verbb/formie/releases/tag/1.4.13)
